inference-engine/tests/ie_test_utils/functional_test_utils/layer_test_utils.cpp

```diff
@@ -92,11 +99,14 @@ void LayerTestsCommon::Infer() {
 
     for (const auto &input : cnnNetwork.getInputsInfo()) {
         const auto &info = input.second;
-
         auto blob = GenerateInput(*info);
         inferRequest.SetBlob(info->name(), blob);
         inputs.push_back(blob);
     }
+    if (configuration.count(InferenceEngine::PluginConfigParams::KEY_DYN_BATCH_ENABLED)) {
+        auto bathSize = cnnNetwork.getInputsInfo().begin()->second->getTensorDesc().getDims()[0] / 2;
```

Comment:
You are using different formula for half batch calculation. 
Previously in code : 
`batchSize = resize > 1 ? resize/ 2 : 1;`